Dashboard Data Transfer/Transfer scripts/transfer_admissions.R

```diff
@@ -3,17 +3,17 @@
 
 ##### 5. Admissions
 
-adm_path <- "/conf/PHSCOVID19_Analysis/RAPID Reporting/RAPID CHI & DOA"
+adm_path <- "/conf/PHSCOVID19_Analysis/RAPID Reporting/Daily_extracts"
 
-i_adm <- read_all_excel_sheets(glue("{adm_path}/Pubs {format(report_date-2, format='%d%m')}/12_Admissions_Positives_{format(report_date-2, format='%Y-%d-%m')}.xlsx"))
+i_adm <- read_all_excel_sheets(glue("{adm_path}/12_Admissions_Positives_provisional.xlsx"))
 
 read_rds_with_options <- create_loader_with_options(readRDS)
-i_chiadm <- read_rds_with_options(glue("{adm_path}/Pubs {format(report_date-2, format='%d%m')}/CHI_Admissions_Positives_{format(report_date-2, format='%Y-%d-%m')}.rds"))
+i_chiadm <- read_rds_with_options(glue("{adm_path}/CHI_Admissions_Positives_provisional.rds"))
 
 o_adm_simd <- read.csv(glue("{output_folder}/Admissions_SIMD.csv"), header = TRUE, stringsAsFactors = FALSE, check.names=FALSE)
 
 # Filter CHI and 12 files down to last Tuesday
-i_chiadm %<>%filter(admission_date <= (report_date - 8))
+i_chiadm %<>% filter(admission_date <= (report_date - 8))
```

Comment:
```suggestion
i_chiadm %<>% filter(admission_date <= (report_date - 3))
```

Reply:
@rubenv95 was this what you mean?

Reply:
Yes thanks, can you also please add a "p" for provisional to the latest week of data for hospital admissions data tables?